### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by suve** [theZiz/aha#53]
Since recently there's been quite a lot of new features introduced, I think it'd be great to bump the version number and have a new release.

**Comment by theZiz** [comment]
Yes, a new release is planned, I just wanted to wait until the discussion around the new features are finished and everything worked fine :smiley:
